lec06-pop-models.Rmd

```diff
@@ -2,3 +2,298 @@
 title: "Qualitative review of population models"
 ---
 
+
+## Lesson preamble
```

Comment:
Maybe to be consistent with how Joel has done it, put the lesson preamble all in blockquotes